### PR TITLE
Exercises importer now skips exercises with no changes

### PR DESCRIPTION
**Pull request by Dantemss**

### CODE REVIEW

**Comment by lakshmivyas** [comment]
@Dantemss: Sorry about the late epiphany.   Shouldn't the code for checking old vs new attributes go into exercises core?   If someone makes changes in the UI, reverts and saves, we should do the same thing, no?

**Comment by Dantemss** [comment]
I think the plan on the UI side is to not allow changes unless you specifically request a new version first.

**Comment by lakshmivyas** [comment]
@Dantemss - regardless of how the UI is designed, shouldn't the exercises code behave in this manner - when there is no change in content, new version is not created?

**Comment by Dantemss** [comment]
I didn't mean just the UI, I meant the API's for the UI's use... creating a new version outside of the import process will be a separate API. Changes will not be allowed to published exercises.

File: app/routines/exercises/import/row_importer.rb [hunk 1]
```diff
@@ -176,9 +200,30 @@ def perform_row_import(row,index)
         ex.save!
         list.list_exercises << le
 
-        Rails.logger.info "Imported #{index} exercise(s) - Current uid: #{ex.uid} - New #{
-                            'version of existing ' unless latest_exercise.nil?
-                          }exercise"
+        skipped = false
+        unless latest_exercise.nil?
+          old_attributes = latest_exercise.association_attributes(
```
Comment (jpslav) [inline app/routines/exercises/import/row_importer.rb]: @Dantemss is a rails monster

**Comment by jpslav** [comment]
Looks good.

**Comment by lakshmivyas** [comment]
@Dantemss - makes sense.  Thanks.

**Pull request merged**